packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -745,31 +745,34 @@ const managerBehavior = {
 };
 
 const collateralBehavior = {
-  /** @param {MethodContext} context */
-  makeVaultInvitation: ({ state, facets: { self } }) => {
-    const { zcf } = provideEphemera(state.collateralBrand);
+  makeVaultInvitation() {
```

Comment:
nit: it seems like the `const { state, facets } = this;` idiom would have made a smaller / easier to read diff here.

Reply:
fair. I tended to use that but when there was only one reference sometimes it seemed like the code would read better in the long term by inlining.